lib/modules/manager/composer/readme.md

```diff
@@ -1 +1,11 @@
 Extracts dependencies from `composer.json` files, and keeps the associated `composer.lock` file updated too.
+
+If [VCS repositories](https://getcomposer.org/doc/05-repositories.md#vcs) are used Renovate needs a hint via the `name` property, which has to match the relevant package. E.g. the package `acme/foo` would need an entry in [repositories](https://getcomposer.org/doc/04-schema.md#repositories) similar to the following
```

Comment:
```suggestion
If you use [VCS repositories](https://getcomposer.org/doc/05-repositories.md#vcs) then Renovate needs a hint via the `name` property, which must match the relevant package.
For example, the package `acme/foo` would need an entry in [repositories](https://getcomposer.org/doc/04-schema.md#repositories) similar to the following:
```

- Put the reader into the sentence by using "you".
- use `then` as connecting word
- Replace `e.g.` with `For example,`
- Add `:` at the end of the second sentence

Reply:
thank you, great tips as always!